documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
```

Comment:
@jennijuju is this going to be a link to a notion?

Reply:
I think it should be a list of issues in this section.

Reply:
Note that this should ideally be very few items for minor releases -- I really wanna move away from blocking releases on stuff that's "almost ready".